src/elements/emby-scroller/emby-scroller.js

```diff
@@ -17,9 +17,32 @@ import 'css!./emby-scroller';
 
     function initCenterFocus(elem, scrollerInstance) {
         dom.addEventListener(elem, 'focus', function (e) {
-            const focused = focusManager.focusableParent(e.target);
-            if (focused) {
-                scrollerInstance.toCenter(focused);
+            if (e.explicitOriginalTarget.parentNode === e.target.parentNode) {
```

Comment:
`explicitOriginalTarget` isn't supported in chrome?

Reply:
Ah, shit... I need the previous target to get this to work... Maybe `relatedTarget` could work...

Reply:
@Camc314 did you want to adopt this pull request?